.github/workflows/dev_build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Dev Build
+
+on:
+  push:
+    branches:
+      - test/*
+
+jobs:
+  build-linux:
```

Comment:
you could do this as a `matrix` https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow